src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    public Ball getRandomBall() {
```

Comment:
Pay attention to access modifiers. Methods should explicitly declare their visibility for better code readability and encapsulation. In this case, `getRandomBall` can be declared as `public`.